TwilioVerifyDemo/TwilioVerifyDemo/Settings/SettingsViewController.swift

```diff
@@ -0,0 +1,135 @@
+//
+//  SettingsViewController.swift
+//  TwilioVerifyDemo
+//
+//  Created by Alejandro Orozco Builes on 16/12/21.
+//  Copyright © 2021 Twilio. All rights reserved.
+//
+
+import UIKit
+import TwilioVerifySDK
+
+class SettingsViewController: UIViewController, UITableViewDataSource, UITableViewDelegate {
+  
+  let storage = SecureStorage(accessGroup: nil)
+
+  enum AppSetting: String, CaseIterable {
+    case clearStorage
+    case currentVersion
+    case useAccessGroup
+
+    var description: String {
+      switch self {
+        case .clearStorage:
+          return "Clear Storage"
+        case .currentVersion:
+          return "Storage Version"
+      case .useAccessGroup:
+        return "Use access group"
+      }
+    }
+
+    var type: SettingType {
+      switch self {
+        case .clearStorage: return .bool
+        case .currentVersion: return .int
+        case .useAccessGroup: return .bool
+      }
+    }
+    
+    func toData(value: Any) -> Data {
+      switch type {
+        case .bool: return (value as! Bool).description.data(using: .utf8)!
+        case .int: return withUnsafeBytes(of: (value as! Int).bigEndian) { Data($0) }
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/b791acb3cd9efbf75408116c9fe627d5a3319f09/TwilioVerifyDemo/TwilioVerifyDemo/Settings/SettingsViewController.swift#L43"></span>Force casts should be avoided.<br />
<code>force_cast</code> <code>SettingsViewController.swift:43</code></td>
    </tr>
  </tbody>
</table>